### PR TITLE
feat: add css positioning lab

### LINKED ISSUE #401: chore: add lab project for CSS positioning to frontend cert
The CSS positioning module will a lab for campers to do.
Labs are defined as short projects campers build on their own. It will consist of a blank editor and user stories.
Here is an example for the format:
https://www.freecodecamp.org/learn/the-odin-project/top-build-a-recipe-project/top-build-a-recipe-project

The previous workshop will have been the cat painting project here
https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-intermediate-css-by-building-a-cat-painting/step-80

So you will need to create a project that focuses on absolute positioning.
So maybe another drawing but much smaller and simpler scale then the cat painting? 

## note about the HTML boilerplate

At this point in the cert, campers will have created a few labs and worked on a few workshops.
So we no longer need to force them to write out the HTML boilerplate each time.
So we don't need to create user stories for the HTML boilerplate anymore. 

## Note about adding new images

For adding new images, make sure they are free to use. You will also need to be add that image to our CDN repo here
https://github.com/freeCodeCamp/cdn

Once the new image has been added to the CDN repo, then you will need to reference that cdn link in your project.

If you are not sure about if an image can be used, reach out on staff chat. 

## Acceptance criteria

- [ ] create a new folder of the project in the `frontend-cert/css-projects` directory
- [ ] add your completed project in an `index.html` file and `styles.css` file in that new folder
- [ ] create a markdown file for your proposed user stories. You should call it `user-stories.md` and place it in your new project folder

### PR DESCRIPTION
**Pull request by Dario-DC**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #401

The drawing is pretty geometric and straightforward. I'm not sure if I should add other elements to it or it will be fine as is.

image

### CODE REVIEW

File: frontend-cert/css-projects/rainbow-paint/user-stories.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+1. You should have a `div` element with the `id` of `container`.
+
+1. Your `#container` should have 9 `div` elements nested inside it
```
Comment (moT01) [inline frontend-cert/css-projects/rainbow-paint/user-stories.md:3]:
```suggestion
1. Your `#container` should have nine `div` elements nested inside it
```

I think typically, numbers are spelled out in these cases.

File: frontend-cert/css-projects/rainbow-paint/user-stories.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+1. Each `div` element should have `position` set to `absolute`.
+
+1. Each `div` element should have `border-radius` set to `50%` except for `#cover`.
+
+1. You should target `#container` and set its `width` and `height` properties to `350px` and center the element in the viewport.
+
+1. `#red`, `#orange`, `#yellow`, `#green`, `#blue`, `#indigo`, `#violet`, and `#white` should have the color of their `id` and the `width` and `height` properties of each of them should be `15px` less than the previous one, starting from `250px`.
+
+1. You should use absolute positioniing to center `#red` respect to its parent, and each of the remaining `div` elements (except `#cover`) respect to the previous element.
```
Comment (moT01) [inline frontend-cert/css-projects/rainbow-paint/user-stories.md:15]:
```suggestion
1. You should use absolute positioning to center `#red` respect to its parent, and each of the remaining `div` elements (except `#cover`) respect to the previous element.
```

**Review by moT01 — COMMENTED** [review]
I think this looks pretty good @Dario-DC 👍

I would have liked to see some use of other position values - relative and fixed are pretty common. I think it's fine without those though.

**Comment by Dario-DC** [comment]
> I would have liked to see some use of other position values - relative and fixed are pretty common. I think it's fine without those though.

If we want, we can add a cloud or another object. The issue is saying "a lab focused on absolute positioning", so I wasn't thinking of using other options.

Let me know if you think I should modify it.

**Review by moT01 — APPROVED** [review]
> a lab focused on absolute positioning

I missed that part. Yea, lets go with is then. I'm happy with it if you are @Dario-DC - feel free to merge it.

File: frontend-cert/css-projects/rainbow-paint/styles.css [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    left: 50px;
+}
+
+#orange {
+    background-color: orange;
+    width: 235px;
+    height: 235px;
+    left: 57.5px;
+    top: 57.5px;
+
```
Comment (Dario-DC) [inline frontend-cert/css-projects/rainbow-paint/styles.css:36]:
```suggestion
```

**Pull request merged**